style.css

```diff
@@ -577,4 +583,607 @@
     background: #2230d2;
     color: #fff;
   }
+
+  #mobile {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) {
+  body {
+    background: #e5e5e5;
+    width: 100%;
+    overflow-x: hidden;
+  }
+
+  .hide {
+    background-color: #fff;
+    padding: 10px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    position: sticky;
+    top: 0;
+    margin-top: -1%;
+    z-index: 999;
+    margin-left: -1%;
+    width: 100%;
+    box-shadow: 0 10px 10px -10px rgba(0, 0, 0, 0.5);
+    border: none;
+  }
+
+  .logos {
+    display: flex;
+    flex-direction: row;
+    margin-left: 50%;
+    text-align: center;
+    flex-wrap: nowrap;
+    width: 76px;
+    height: 20px;
+    font-family: 'poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    color: #6070ff;
+    padding-top: 1rem;
+  }
+
+  #portfolio {
+    margin-left: 250%;
+  }
+
+  .navlist {
+    font-family: 'poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #325080;
+  }
+
+  .hide-items {
+    display: flex;
+    flex-direction: row;
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    flex-wrap: nowrap;
+  }
+
+  ul li a {
+    display: flex;
+    flex-direction: row;
+    padding: 10px;
+    text-decoration: none;
+    color: #333;
+  }
+
+  .header {
+    display: none;
+  }
+
+  h1 {
+    width: 57.5rem;
+    height: 7.6rem;
+    font-family: 'poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    padding-top: 20%;
+  }
+
+  .Intro {
+    width: 100%;
+    height: 40rem;
+    top: 1px;
+    background: #fff;
+    align-items: center;
+    margin-top: -9%;
+    margin-left: -1%;
+    margin-right: 3%;
+    background-image: url(head-back.svg);
```

Comment:
- [ ] kindly consider making sure that this background is all visible in your intro section like it is on the Figma design to make your work an exact duplicate of the given one 
- your design 
<img width="1440" alt="Screen Shot 2023-06-09 at 09 15 00" src="https://github.com/Omwomo/Microverse-web-portfolio-project/assets/60146030/d98ee4ea-7621-4445-92c0-72afabf86204">

- Figma design 
<img width="735" alt="Screen Shot 2023-06-09 at 09 15 52" src="https://github.com/Omwomo/Microverse-web-portfolio-project/assets/60146030/e1394c7f-777e-4031-ace8-150a47f0ef73">